.github/workflows/test-only.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         uses: actions/setup-java@v4
         with:
           distribution: 'zulu'
-          java-version: '11'
+          java-version: '17'
```

Comment:
I'm seeing demo projects still have release set to 11, should I revert this @slandelle?